daemon/buildinfo.c

```diff
@@ -200,8 +200,67 @@
 
 #define FEAT_YES_NO(x) ((x) ? "YES" : "NO")
 
+// coverity[ +tainted_string_sanitize_content : arg-0 ]
+static inline void coverity_remove_taint(char *s)
+{
+    (void)s;
+}
+
+void read_install_type(void) {
+    char *filepath;
+    filepath = mallocz(sizeof(char) * (strlen(netdata_configured_user_config_dir) + strlen(".install-type") + 2));
+    sprintf(filepath, "%s/%s", netdata_configured_user_config_dir, ".install-type");
+    FILE *fp = fopen(filepath, "r");
+
+    if(fp) {
+        char *line = mallocz(sizeof(char) * (200 + 1));
+        while (fgets(line, 200, fp) != NULL) {
```

Comment:
@Ferroin consider using `fscanf` and be very strict about what we are accepting as a valid `.install_type` file because we are the ones generating that file in the 1st place.